extensions/amp-analytics/0.1/sandbox-vars-whitelist.js

```diff
@@ -48,4 +48,5 @@ export const SANDBOX_AVAILABLE_VARS = {
   'FIRST_CONTENTFUL_PAINT': true,
   'FIRST_VIEWPORT_READY': true,
   'MAKE_BODY_VISIBLE': true,
+  'HTML_ATTR': true,
```

Comment:
is this change needed?

Reply:
Removed locally and tested, no change. Will leave out.